datadog_cluster_agent/manifest.json

```
@@ -0,0 +1,34 @@
+{
+  "display_name": "Datadog Cluster Agent",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "datadog_cluster_agent",
+  "metric_prefix": "datadog.cluster_agent.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Track Datadog Cluster Agent's metrics",
+  "guid": "2b116278-fa9c-4efd-a250-fa6baf209607",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog Cluster Agent Integration",
+  "categories": [
+    "containers"
+  ],
+  "type": "check",
+  "is_public": true,
```

I don't think this should be `true` until after the next Agent release, since otherwise the integration will show up to customers without being available in the latest Agents. Usually we switch this to `true` just before the QA cycle starts (so that the integration is published along with the first RCs).

```suggestion
  "is_public": false,
```